cumulus/pallets/xcmp-queue/src/lib.rs

```diff
@@ -664,13 +711,17 @@ impl<T: Config> OnQueueChanged<ParaId> for Pallet<T> {
 		let suspended = suspended_channels.contains(&para);
 
 		if suspended && fp.ready_pages <= resume_threshold {
-			Self::send_signal(para, ChannelSignal::Resume);
+			// If the resuming fails then it is not critical. We will retry in the future.
```

Comment:
Why can the resuming fail? Why is it not critical?

Reply:
If it fails here then the channel is either broken or something else with HRMP delivery does not work.  
Best we can do is to retry, so from the perspective here "not critical" means that we keep buffering messages and not dropping them. Amending the comment.

Reply:
Hm the code before is not so good either. Actually it should not remove the channel from the `InboundXcmpSuspended` in this case. Will change.

Reply:
Changed it to better handle the error. "not critical" in the sense that we dont have to invalidate the candidate for this (=panic).